pypeerassets/transactions.py

```diff
@@ -4,6 +4,7 @@
 from math import ceil
 from time import time
 
+from btcpy.constants import Constants
```

Comment:
linter reports that this import is not used.

Reply:
i'll double check it, i bet the linter is right :)

what linter/command do you run? i should probably use it too

Reply:
flake-8